data-processing/tests/test_extract_definitions.py

```diff
@@ -137,6 +240,34 @@ def test_extract_nicknames_symbols_parentheses():
     assert nickname0.term_text == r"\pi"
     assert nickname0.definition_text == "policy"
 
+@pytest.mark.slow
+def test_model_extracts_nickname_symbol_filter(model: DefinitionDetectionModel):
+    prediction_type = 'SYM-NICK'
+    features = model.featurize("The agent acts with SYMBOL SYMBOL")
+    intents, slots, _ = model.predict_batch([features], prediction_type)
+    assert intents[0] == 0
+    assert slots[0] == ["O", "O", "O", "O", "O", "O"]
+
+
+@pytest.mark.slow
+def test_extract_symbol_nickname_pairs_case_4():
```

Comment:
nit pick: give test more descriptive name indicating the specific behavior being tested